src/ext/language_tools.js

```diff
@@ -51,7 +53,8 @@ var snippetCompleter = {
                     caption: caption,
                     snippet: s.content,
                     meta: s.tabTrigger && !s.name ? s.tabTrigger + "\u21E5 " : "snippet",
-                    type: "snippet"
+                    type: "snippet",
```

Comment:
type here was serving the same purpose as completerId

Reply:
Removed in new version